wolfhsm/wh_comm.h

```diff
@@ -35,9 +30,9 @@
  * DATA_LEN bytes.
  */
 enum {
-    WOLFHSM_COMM_HEADER_LEN = 8,
-    WOLFHSM_COMM_DATA_LEN = 1280,
-    WOLFHSM_COMM_MTU = (WOLFHSM_COMM_HEADER_LEN + WOLFHSM_COMM_DATA_LEN)
+    WM_COMM_HEADER_LEN = 8,    /* whCommHeader */
```

Comment:
Is this a typo in the prefix? Shouldn't it be `WH` not `WM`?:
```suggestion
    WH_COMM_HEADER_LEN = 8,    /* whCommHeader */
```
If so, you will need to go through and update all the references

Reply:
Sigh.  Fixed in next push.